docker/build.sh

```diff
@@ -54,7 +54,6 @@ function make_libgmp {
   (
     cd "${LIBGMP_INSTALL}" || return
     ./configure \
-      --build=x86_64-linux-gnu \
```

Comment:
```shell
if [[ "$kernel" = "aarch64" ]] || [[ "$machine" == "arm64" ]]
```
Maybe "Darwin" is not strong enough, because someone might develop it inside a virtual machine.

Reply:
+1 to only change that when a m1 is detected